src/data/articles/beginners-guide-to-programming.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Beginners Guide to Programming
+author: _xsaratoninx_
+date: '2024-05-07'
+description: How to start teaching yourself to write code, advice from those who've done it before.
+---
+# A Beginner's Guide to Programming
```

Comment:
The title will get automatically injected based on the parameters above, so you'll want to remove this else it'll be there twice!